config/environments/production.rb

```diff
@@ -1,6 +1,15 @@
 # frozen_string_literal: true
 
 Rails.application.configure do
+  config.paperclip_defaults = {
+    storage: :s3,
+    s3_credentials: {
+      bucket: ENV.fetch('S3_BUCKET_NAME'),
+      access_key_id: ENV.fetch('AWS_ACCESS_KEY_ID'),
+      secret_access_key: ENV.fetch('AWS_SECRET_ACCESS_KEY'),
+      s3_region: ENV.fetch('AWS_REGION')
```

Comment:
We should have file like `.env.example` with examples

```
export S3_BUCKER_NAME=<placeholder>
...
```

This will make initial setup of the app for new developers easier.